_episodes/13-conditionals.md

```diff
@@ -24,58 +24,65 @@ keypoints:
     *   Body containing one or more statements is indented (usually by 4 spaces)
 
 ~~~
-mass = 3.54
-if mass > 3.0:
-    print(mass, 'is large')
+animal = "dog"
+if animal == "dog":
+    print(animal, 'is fluffy')
 
-mass = 2.07
-if mass > 3.0:
-    print (mass, 'is large')
+animal = "snake"
+if animal == "dog":
+    print (animal, 'is fluffy')
 ~~~
 {: .language-python}
 ~~~
-3.54 is large
+dog is fluffy
 ~~~
 {: .output}
 
 ## Conditionals are often used inside loops.
 
 *   Not much point using a conditional when we know the value (as above).
 *   But useful when we have a collection to process.
-
+*   We can also use the in operator on strings directly.
 ~~~
-masses = [3.54, 2.07, 9.22, 1.86, 1.71]
-for m in masses:
-    if m > 3.0:
-        print(m, 'is large')
+animals = ["aardvark", "bat", "cat", "dog"]
+acount = 0
+for animal in animals:
+   for letter in animal:
+      if letter == 'a':
+         acount += 1
```

Comment:
When I taught this recently it took a while for people to get used to the idea of modifying that accumulator (acount in this case).  So I think it would be better to write out the `acount +` part instead of using `+=` notation since this shows more explicitly what is going on.
```suggestion
         acount = acount + 1
```